hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1409,19 +1409,39 @@ message SetS3SecretResponse {
 message TenantInfo {
     optional string tenantId = 1;
     optional string bucketNamespaceName = 2;
-    optional string accountNamespaceName = 3;
-    optional string userPolicyGroupName = 4;
-    optional string bucketPolicyGroupName = 5;
+    repeated string policyNames = 3;
```

Comment:
Should we track the roles associated with each tenant here?

Reply:
We could. Should we do that?
We didn't have tenant -> roleNames mapping before.

Unless:
1. we are intending to store some roleIndex -> roleName mapping in TenantInfo (TenantState) and use roleIndex in ExtendedAccessIdInfo to reduce redundancy. e.g. tenant1-UserRole = index 0, tenant1-AdminRole = index 1.
2. it is required to iterate all role names assigned in this tenant quickly
3. the Ranger sync thread would need it when deleting a tenant

(3) might be valid? @prashantpogde